LDAR_Sim/src/programs/program.py

```diff
@@ -133,7 +133,7 @@ def split_methods(self, methods: dict) -> Tuple[dict, dict]:
         follow_up_methods: dict = {}
         other_methods: dict = {}
         for method, properties in methods.items():
-            if properties[Method.FOLLOW_UP_ACCESSOR]:
+            if properties[Method_Params.IS_FOLLOW_UP]:
```

Comment:
Nit: We should be consistent about using pdc.Method params vs importing method params